plugins/foreman_discovery/18.0/index.md

```diff
@@ -0,0 +1,946 @@
+---
+layout: plugin
+pluginname: foreman_discovery
+title: Foreman Discovery Manual
+version: 18.0
+# versions for matrix and snippets
+# (use short version for imgver e.g. 3.0)
+pluginver: 18.0.0
+proxyver: 1.0.4
+imgver: 3.5
+cliver: 1.0.2
+# uncomment to show warning box for an old release
+#warning: old
+# uncomment to show development version warning
+#warning: unreleased
+---
+
+# 1. Foreman Discovery {{ page.version }} manual
+
+This plugin enables Foreman to do automatic bare-metal discovery of unknown
+nodes on the provisioning network. This is often referenced as
+Metal-as-a-Service or simply MaaS. New nodes self-register into Foreman and
+upload facts collected by Facter (serial id, network interfaces, memory,
+disks). The registered nodes show up on Discovered Hosts page and provisioning
+can be initiated either manually (via UI/CLI or API) or automatically via
+predefined Discovery Rules.
+
+All communication can be optionally passed through Smart Proxy which has
+direct access both to the provisioning network and Foreman instance:
+
+![Discovery Communication Workflows](/static/images/diagrams/discovery_comm.png)
+
+## 1.1 Release Notes
+
+To read release announcements for discovery plugin, smart proxy plugin and
+image, visit the [Release Announcement category](https://community.theforeman.org/search?q=%23release-announcements%20tags%3Adiscovery)
+on our site.
+
+# 2. Installation
+
+There are three different components to install: Foreman Discovery plugin,
+Smart Proxy Discovery plugin (aka Smart Proxy Discovery) and Foreman
+Discovery Image.
+
+## 2.1 Foreman Discovery plugin
+
+Foreman Discovery plugin is the main component that needs to be installed.
+
+### 2.1.1 Installer (recommended)
+
+The foreman-installer is able to automatically install the package and proceed
+with all necessary steps. To do this, re-run the installer with the following
+option:
+
+    # foreman-installer --enable-foreman-plugin-discovery
+
+Important note: Executing foreman-installer will re-deploy all foreman-related
+configuration files. In case some changes have been made, proceed with manual
+download described below.
+
+### 2.1.2 Manual installation
+
+To install Foreman Discovery plugin, do the following
+
+    # yum install tfm-rubygem-foreman_discovery
```

Comment:
the package is called `rubygem-foreman_discovery` on EL8, `tfm-…` is only correct for EL7.

We don't have packages for Fedora anymore.